hybrid/basic_ddc.c

```diff
@@ -0,0 +1,31 @@
+/***
+ * This programs shows a very basic level
+ * of comportmentalisation. It first `malloc`
```

Comment:
s/comportmentalisation/compartmentalisation/

Reply:
Fix in 239cfd0.